perpendicular-flap/solid-fenics/requirements.txt

```
@@ -1,4 +1,4 @@
-fenicsprecice~=2.0
+fenicsprecice==2.2.0rc1
```

```suggestion
fenicsprecice>=2.2
```
As soon as https://github.com/precice/fenics-adapter/pull/184 is merged and the new release is available.